keyboards/40percentclub/i75/rules.mk

```
@@ -0,0 +1,23 @@
+# Build Options
+#   change yes to no to disable
+#
+BOOTMAGIC_ENABLE = no       # Virtual DIP switch configuration(+1000)
```

May want to consider using `lite` here.  Entirely optional though.

```suggestion
BOOTMAGIC_ENABLE = lite     # Virtual DIP switch configuration(+1000)
```

---

Done